lib/Backend/Commands.hs

```diff
@@ -87,9 +87,16 @@ viewCmd config (ViewOptions (QualifiedPath backendNameMb path) fieldNameMb) = do
 
 createCmd
   :: forall r
-   . (Member BackendEffect r, Member (Error CofferError) r, Member (Embed IO) r, Member (Error CreateResult) r)
+   . ( Member BackendEffect r
+     , Member (Error CofferError) r
+     , Member (Embed IO) r
+     , Member (Error CreateResult) r
+     )
```

Comment:
I think we can use [Members](https://hackage.haskell.org/package/polysemy-1.7.1.0/docs/Polysemy.html#t:Members) instead of `Member` here (and in some other places)

Reply:
Good catch! Done